data/campaigns/Eastern_Invasion/_main.cfg

```diff
@@ -16,6 +16,7 @@
         >>
 [/lua]
 
+{resources}
```

Comment:
What? No. This should go in the toplevel `_main.cfg`, not in EI.

Reply:
Ok, done